backend/src/services/worker/worker.service.ts

```diff
@@ -220,26 +185,6 @@ class Worker {
       });
     }
 
-    const instances = await this.#instanceRepository.getByWorkerId(id);
-    const spaces = await this.#spaceRepository.getByWorkerId(id);
-    const slcFunctions = await this.#slcFunctionRepository.getByWorkerId(id);
-
-    await Promise.all(
```

Comment:
and how we will delete all user's data?

Reply:
And I think it will invoke an error in the DB